tools/rack/rack.plugin/META-INF/MANIFEST.MF

```diff
@@ -90,7 +91,11 @@ Bundle-ClassPath: .,
  lib/json-simple.jar,
  lib/opencsv.jar,
  lib/sparqlGraphLibrary.jar,
- lib/viz.js-graphviz-java.jar
+ lib/viz.js-graphviz-java.jar,
+ lib/javafx-swing.jar,
+ lib/javafx-swing-linux.jar,
+ lib/javafx-swing-mac.jar,
+ lib/javafx-swing-win.jar
```

Comment:
Please keep all the jars sorted alphabetically.  This helps maintainers check for missing jars which are in the lib directory but not on the bundle's classpath (note we do omit some jars from the lib directory on purpose, but still).

Reply:
Fixed.